specification/cognitiveservices/data-plane/AnomalyDetector/preview/v2.0-preview/MultidimensionalRootCauseAnalysis.json

```diff
@@ -0,0 +1,2031 @@
+    "version": "2.0-preview"
+  },
+  "paths": {
+    "/rootCauseAnalysis/multiDimension/dataSets": {
+      "post": {
+        "tags": [
+          "MultiDimRCADataSet"
+        ],
+        "summary": "Create a multidimensional root cause analysis data set",
+        "operationId": "CreateMultiDimRCADataSet",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "create a multidimensional root cause analysis data set request",
+            "schema": {
+              "$ref": "#/definitions/CreateMultiDimRCADatasetRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Success",
+            "headers": {
+              "Location": {
+                "description": "Location of the newly created resource.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Client error or server error (4xx or 5xx)",
+            "schema": {
+              "$ref": "#/definitions/ErrorCode"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a multidimensional root cause analysis data set": {
+            "$ref": "./examples/CreateMultiDimRCADataSet.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "MultiDimRCADataSet"
+        ],
+        "summary": "List multidimensional root cause analysis data set",
+        "operationId": "ListMultiDimRCADataSet",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "name",
+            "description": "Multidimensional root cause analysis data set name",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "$skip",
```

Comment:
DO NOT use the "$" prefix on OData query parameter names -- the Azure standard is to drop the "$".